server/plugin.go

```diff
@@ -29,7 +30,8 @@ import (
 )
 
 const (
-	GitlabTokenKey       = "_gitlabtoken"
+	GitlabUserInfoKey    = "_userinfo"
+	GitlabUserTokenKey   = "_usertoken"
```

Comment:
I think changing these key names is the right move since we are going to force users to reconnect anyway. This results in persisting stale `UserInfo/Token` structs in their old entries forever, but it seems more appropriate than having the wrong data type stored in `_gitlabtoken` or a similar situation